src/mlpack/core/tree/binary_space_tree/binary_space_tree.hpp

```diff
@@ -384,9 +384,15 @@ class BinarySpaceTree
    */
   ElemType FurthestDescendantDistance() const;
 
+  //! Modify the furthest possible descendant distance.
+  ElemType& FurthestDescendantDistance() { return furthestDescendantDistance; }
```

Comment:
It seems like this change is making a number of tests to fail.

Reply:
What have you tried to debug this?